tests/end2end/playwright/popup.spec.js

```diff
@@ -246,184 +246,189 @@ test.describe('Raster identify',
         });
     });
 
-test.describe('Popup', () => {
+test.describe('Popup',
+    {
+        tag: ['@readonly'],
+    },() => {
 
-    test.beforeEach(async ({ page }) => {
-        const url = '/index.php/view/map/?repository=testsrepository&project=popup';
-        await gotoMap(url, page);
-    });
+        test('click on the shape to show the popup', async ({ page }) => {
+            const project = new ProjectPage(page, 'popup');
+            await project.open();
 
-    test('click on the shape to show the popup', async ({ page }) => {
-        // When clicking on triangle feature a popup with two tabs must appear
-        await page.locator('#newOlMap').click({
-            position: {
-                x: 510,
-                y: 415
-            }
-        });
-        await expect(page.locator('#newOlMap #liz_layer_popup')).toBeVisible();
-        await expect(page.locator('#newOlMap #liz_layer_popup_contentDiv > div > div > div > ul > li > button.active')).toBeVisible();
-        await expect(page.locator('#newOlMap #liz_layer_popup_contentDiv > div > div > div > ul > li:nth-child(2) > button')).toBeVisible();
-    });
+            let getFeatureInfoPromise = project.waitForGetFeatureInfoRequest();
 
-    test('changes popup tab', async ({ page }) => {
-        // When clicking `tab2`, `tab2_value` must appear
-        await page.locator('#newOlMap').click({
-            position: {
-                x: 510,
-                y: 490
-            }
+            // When clicking on triangle feature a popup with two tabs must appear
+            await project.clickOnMap(510, 415);
+            await getFeatureInfoPromise;
+            await expect(page.locator('#newOlMap #liz_layer_popup')).toBeVisible();
```

Comment:
I guess in your file `tests/end2end/playwright/popup.spec.js`, you can do a search& replace `page.locator('#newOlMap` with `project.map.locator(` for all other occurrences.